code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -296,6 +296,7 @@
 				recycler.use_power(500)
 				X.monkeys += recycler.cube_production
 				X.monkeys = round(X.monkeys, 0.1)		//Prevents rounding errors
+				M.moveToNullspace()						// Prevents monkey staying there giving unlimited monkeys if the gc is busy
 				qdel(M)
```

Comment:
`qdel` -> `Destroy` -> `moveToNullspace` already handles this. The issue is probably this runtime:
```
[2020-08-08 01:53:55.217] runtime error: Cannot execute null.Cut().
 - proc name: Destroy (/mob/living/Destroy)
 -   source file: living.dm,36
 -   usr: Raiden Noid (/mob/living/carbon/human)
 -   src: the monkey (300) (/mob/living/carbon/monkey)
 -   usr.loc: the floor (133,64,2) (/turf/open/floor/plasteel)
 -   src.loc: the reinforced floor (127,58,2) (/turf/open/floor/engine)
 -   call stack:
 - the monkey (300) (/mob/living/carbon/monkey): Destroy(0)
 - the monkey (300) (/mob/living/carbon/monkey): Destroy(0)
 - the monkey (300) (/mob/living/carbon/monkey): Destroy(0)
 - qdel(the monkey (300) (/mob/living/carbon/monkey), 0)
 - Recycle Monkeys (/datum/action/innate/monkey_recycle): Activate()
 - Recycle Monkeys (/datum/action/innate/monkey_recycle): Trigger()
 - Recycle Monkeys (/obj/screen/movable/action_button): Click(null, "mapwindow.map", "icon-x=9;icon-y=17;left=1;scre...")
 - Xommel (/client): Click(Recycle Monkeys (/obj/screen/movable/action_button), null, "mapwindow.map", "icon-x=9;icon-y=17;left=1;scre...")
```

Reply:
The line of code for that is line 36 living.dm, as per the runtime.
QDEL_LIST(diseases)

In living_defines.dm, `var/list/diseases = list() /// list of all diseases in a mob` clearly shows that diseases is not a lazy list and we should not expect it to be null, so there must be a code path setting it null somewhere.

Reply:
It's actually a lazylist now. https://github.com/tgstation/tgstation/blob/09ef529108850a61711d9c84be26e21a233af5bb/code/modules/mob/living/living_defines.dm#L123

Reply:
Humm I will mess with it localy.  I thought Destroy moves it to null space but I thought there was some GC delay